src/init.cc

```diff
@@ -8,16 +8,19 @@
 #include "heapdiff.hh"
 #include "memwatch.hh"
 
+using namespace v8;
+
 extern "C" {
-    void init (v8::Handle<v8::Object> target)
+    void init(Local<Object> exports)
```

Comment:
Recommend reverting most of this stuff - keep the diffs from mainline as minimal as possible to ease pulling other changes through in the future.  AFAICT, these changes are not necessary.

Reply:
This could be `v8::Local`.